src/cpp/session/modules/SessionProfiler.R

```diff
@@ -13,4 +13,54 @@
 #
 #
 
+.rs.addJsonRpcHandler("start_profiling", function(profilerOptions)
+{
+   tryCatch({
+      tempPath <- .Call("rs_profilesPath")
+      if (!dir.exists(tempPath)) {
```

Comment:
Unfortunately `dir.exists` is a relatively new addition to R. Here's a version which is compatible with older versions:

https://github.com/rstudio/rmarkdown/blob/aac512506deb3e133fbf0251b1693835f7b677c5/R/util.R#L117-L119